### DISCUSSION TITLE
Sandbox Version link not working on Serializers Tutorials Page in docs

### DISCUSSION THREAD

**Discussion by hiteshbandhu** [encode/django-rest-framework#9357] [Potential Issue]
![image](https://github.com/encode/django-rest-framework/assets/90546635/9b2cbe6e-4481-4e73-93e6-528158ed934f)

#### The link to the working prototype is not available, the heroku website throws an error while accessing it.

**Comment by TGoddessana** [comment]
It's been discussed here( #8887 ), but we don't seem to have settled on which solution to choose - maybe deleting that part of the documentation would be a good option?

**Reply by lovelydinosaur** [reply]
Yes, since we don't have the sandbox running at the moment let's not link to it.

**Reply by TGoddessana** [reply]
Ok, i'll make pr for this.. 👍

**Reply by TGoddessana** [reply]
#9363 ✔️..
